### ISSUE TITLE
Report for multi module gradle project does not exclude project's own group by default

### ISSUE THREAD

**Issue by amimas** [jk1/Gradle-License-Report#85]
Hello,

I've got a multi-module gradle project. I applied this plugin using the plugins block to my root project's `build.gradle`

```gradle
plugins {
	id 'org.sonarqube' version '2.6.2'
	id 'com.github.jk1.dependency-license-report' version '1.1'
}
```

I didn't make any other configurations using `licenseReport` extension as the default configuration in the ReadMe seemed good enough for me. When I run `sh gradlew generateLicenseReport`, I get a lot of error stacktrace like this:
```
Failed to retrieve artifacts for [group:org.amimas.myproject, name:sample-lib, version:2.0-SNAPSHOT, ext:pom]
org.gradle.internal.resolve.ArtifactNotFoundException: Could not find sample-lib.pom (project :sample-lib).
        at org.gradle.internal.resolve.result.DefaultBuildableArtifactResolveResult.notFound(DefaultBuildableArtifactResolveResult.java:27)
        at org.gradle.api.internal.artifacts.ivyservice.projectmodule.ProjectDependencyResolver.resolveArtifact(ProjectDependencyResolver.java:147)
        at org.gradle.api.internal.artifacts.ivyservice.resolveengine.ComponentResolversChain$ArtifactResolverChain.resolveArtifact(ComponentResolversChain.java:122)
        at org.gradle.api.internal.artifacts.ivyservice.ivyresolve.ErrorHandlingArtifactResolver.resolveArtifact(ErrorHandlingArtifactResolver.java:46)
        at org.gradle.api.internal.artifacts.ivyservice.resolveengine.artifact.DefaultArtifactSet$LazyArtifactSource.create(DefaultArtifactSet.java:171)

```
The report is still generated under the root project (`$buildDir/reports/dependency-license`) and it contains 1 liner like the following (one for each gradle subproject), along with actual report of all the dependencies.
```
24. Group: org.amimas.myproject Name: sample-lib Version: 2.0-SNAPSHOT
```

I then added the following to my root project's `build.gradle`
```gradle
licenseReport {
	excludeGroups = ['org.amimas.myproject']
}
```
Now when I run `sh gradlew generateLicenseReport`, I don't get those stacktrace/errors anymore and the report is actually ignoring the group.

Based on the readme, I believe the plugin is supposed to ignore the project's own group by default. Am I missing any configuration? I have the following in my root project's `build.gradle`:

```gradle
group = 'org.amimas.myproject'

allprojects {
    group = rootProject.group
}
```

**Comment by guenhter** [comment]
Very good point. I did it the same way by ignoring the own group but I guess excluding the own group by default might be the better solution.

**Comment by guenhter** [comment]
This should be fixed with https://github.com/jk1/Gradle-License-Report/commit/abe510755580704a00f3cea876aaf3f068044f36

**Comment by amimas** [comment]
Awesome!!! How often does this project release? Does it need certain number of fixes/features before releasing new version?

**Comment by guenhter** [comment]
@amimas A :beer:  for @jk1  will do it I guess :)

**Comment by jk1** [comment]
That's for sure)
Jokes aside, I'll gather some other fixes, add more tests, and release the new version in a couple of days.

**Comment by amimas** [comment]
BTW, thanks again for looking into this. I know I opened several issues over last few days. Sorry about that... 😊 

I'm just too excited to find this plugin and see that it's actually being maintained. Planning on using this in all of my projects.

**Comment by jk1** [comment]
The fix has been released in the scope of 1.2
